### DISCUSSION TITLE
More icons in settings

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#567] [General]
What icons do you want to be added, what preferences/settings, you can send screenshots and icons if you want, I'll adjust them to match the overall design. 

Since I mainly added icons to preferences with subpreferences and to those that have icons (e.g. buttons), some prefs don't have any icons. The challenge arises when determining suitable icons for certain preferences, especially as preferences change over time with additions and removals. Is adding icons necessary for all preferences?"

**Comment by BAXSD** [comment]
YouTube: HDR icon
YouTube Music: Settings icons for all settings

**Reply by anddea** [reply]
You need to be more specific, I don't know what icons could be assigned for left prefs. You can provide exact setting title and what icon could be used.

**Comment by akir45** [comment]
Please add icon for player action buttons `shop` and `thanks`
![Shop icon](https://github.com/anddea/revanced-patches/assets/91464996/03ec6010-ec8c-4bdc-a93b-ac0e34d1ffaf)
![Thanks icon](https://github.com/anddea/revanced-patches/assets/91464996/e8909b53-0cd9-481a-9d54-e42c4654b342)

**Reply by anddea** [reply]
Added, only rewards button is missing

**Reply by akir45** [reply]
I didn't know what the reward button icon was so I didn't add it, but now I know and will send it.![Rewead-icon](https://github.com/anddea/revanced-patches/assets/91464996/b5a12935-130c-4d1c-be42-c040fcca4f2b)

**Reply by anddea** [reply]
Thank you. 

Added this one too. I didn't know how they looked that's why I didn't add them. But if anyone knows then screenshots of how they look are enough, I recreate SVG icons for them.

**Comment by akir45** [comment]
If you have time, add `watch History` and `Enable OPUS codec` icons.

**Reply by anddea** [reply]
I added these icons, let me know if you think they need any refinement.

**Reply by akir45** [reply]
The `Enable OPUS codec` icon might be better to be an English word like the `HDR` and `QUIC procol` icons.

**Reply by anddea** [reply]
I thought so as well, but since Opus has its own logo I just used it but made it Outline.

You can see its logo on Wiki or even more official, their repo. 

https://en.m.wikipedia.org/wiki/Opus_(audio_format)

https://github.com/xiph/opus-logo/tree/master

Do you still think that it would be better as just a word?

**Reply by akir45** [reply]
I'm just a little concerned about it, so no problem if you don't change it.

**Reply by anddea** [reply]
Why concerned? We can also adjust the icon itself if it can be adjusted. It's not a problem, I'll add text icon, I just thought if something has its own icon it's logical to use that instead.

**Reply by akir45** [reply]
I thought it would be better to  unified icon, but I also agree with you that a own icon would be better, theoretically.
